hypha/apply/funds/tests/test_views.py

```diff
@@ -643,6 +755,7 @@ def test_can_screen_applications_in_final_status(self):
         we can still screen it because we are super user
         """
         submission = ApplicationSubmissionFactory(rejected=True)
+        DeterminationFactory(submission=submission, rejected=True, submitted=True)
```

Comment:
Why was this needed here?

Reply:
Reverted this after making the change below